shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -68,13 +75,40 @@ private void chown(AlluxioURI path, String owner, boolean recursive)
     System.out.println("Changed owner of " + path + " to " + owner);
   }
 
+  /**
+   * Changes the owner and group for the directory or file with the path specified in args.
+   * @param path The {@link AlluxioURI} path as the input of the command
+   * @param owner The owner to be updated to the file or directory
+   * @param group The group to be updated to the file or directory
+   * @param recursive Whether change the owner and group recursively
+   */
+  private void chown(AlluxioURI path, String owner, String group, boolean recursive)
+    throws AlluxioException, IOException {
+    SetAttributeOptions options =
+        SetAttributeOptions.defaults().setOwner(owner).setGroup(group).setRecursive(recursive);
+    mFileSystem.setAttribute(path, options);
+    System.out.println("Changed owner:group of " + path + " to " + owner + ":" + group);
+  }
+
   @Override
   public int run(CommandLine cl) throws AlluxioException, IOException {
     String[] args = cl.getArgs();
-    String owner = args[0];
     AlluxioURI path = new AlluxioURI(args[1]);
-    chown(path, owner, cl.hasOption("R"));
-    return 0;
+    Matcher matchUserGroup = USER_GROUP_PATTERN.matcher(args[0]);
+    if (matchUserGroup.matches()) {
+      String owner = matchUserGroup.group("user");
+      String group = matchUserGroup.group("group");
+      chown(path, owner, group, cl.hasOption("R"));
+      return 0;
+    }
+    Matcher matchUserOnly = USER_PATTERN.matcher(args[0]);
+    if (matchUserOnly.matches()) {
+      String owner = matchUserOnly.group("user");
+      chown(path, owner, cl.hasOption("R"));
+      return 0;
+    }
+    System.out.println("The syntax of user/group is invalid.");
```

Comment:
you can print the usage here (`#getUsage()`) to help the user fix their syntax error